pages/components/Cards/cards.css

```diff
@@ -0,0 +1,316 @@
+.cards-display-wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 2px solid var(--white);
+    border-radius: 0;
+}
+
+.icon {
+    width: 2rem;
+    height: 2rem;
+    color: var(--gold1);
+}
+
+.product {
+    display: inline-block;
+    margin-top: 1rem;
+    padding-bottom: 2rem;
+    font-size: 1.6rem;
+    margin-right: 2rem;
+    max-width: 100%;
+}
+
+.product-container {
+    text-align: center;
+    background-color: var(--pink8);
+    color: var(--gold1);
+    box-shadow: 5px 5px 12px #120f1d,
```

Comment:
try to use a variable for box shadow value so that you can use them elsewhere

Reply:
okeiy will add it into variables .thanks!